src/components/RadioGroup/RadioGroup.spec.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import {shallow} from 'component-test-utils-react';
```

Comment:
Please, use `testing-library/react` for tests instead of `component-test-utils-react`

Reply:
Fixed